api/models/model_class_possibilities/Model_id_recursion.py

```diff
@@ -0,0 +1,60 @@
+# for more detail, see class Model in :
+# https://github.com/pallets/flask-sqlalchemy/blob/master/flask_sqlalchemy/model.py
+from typing import Any, List, Type
+
+
+class MyModel(object):
```

Comment:
I would name this to something related to what this class does

Reply:
I have called it `ModelSQL`